CHANGELOG.md

```diff
@@ -30,7 +30,7 @@ Note: as of this release we are now also publishing releases to [Github Containe
 * If you're using the Atlantis Docker image and aren't setting the `--default-tf-version` flag
   then the default version of Terraform will now be 1.0. Simply set the above
   flag to your desired default version to avoid any issues.
-  
+
```

Comment:
[`ghcr.io/runatlantis/atlantis:v0.17.1`](https://github.com/runatlantis/atlantis/pkgs/container/atlantis)

Should be under the `github container registry` heading